README.md

```diff
@@ -40,10 +64,78 @@ Fetch credentials from CyberArk Conjur using the controlling host's Conjur ident
       msg: "{{ lookup('cyberark.conjur.conjur_variable', '/path/to/secret') }}"
 ```
 
-### Author Information
-- CyberArk Business Development Technical Team 
-    - @cyberark-bizdev
-    - @enunez-cyberark
-    - @jimmyjamcabd
-- CyberArk Community and Integrations Team 
-    - @cyberark/community-and-integrations-team
+## Conjur Ansible Role
+This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that identity, secrets can then be retrieved securely using the [Summon](https://github.com/cyberark/summon) tool (installed on hosts with identities created by this role).
+
+### Usage
+The Conjur role provides a method to "Conjurize" or establish the Conjur identity of a remote node with Ansible. The node can then be granted least-privilege access to retrieve the secrets it needs in a secure manner.
```

Comment:
```suggestion
The Conjur role provides a method to establish the Conjur identity of a remote node with Ansible. The node can then be granted least-privilege access to retrieve the secrets it needs in a secure manner.
```
We don't really use `conjurize` as a term in Conjur OSS / DAP (it's a Conjur v4 construct)